tests/failing.llvm

```diff
@@ -1,3 +1,2 @@
-tests/ethereum-tests/BlockchainTests/GeneralStateTests/stBadOpcode/invalidAddr.json
 tests/ethereum-tests/BlockchainTests/GeneralStateTests/stRandom2/randomStatetest649.json
 tests/ethereum-tests/BlockchainTests/GeneralStateTests/stCreate2/CREATE2_HighNonceDelegatecall.json
```

Comment:
Is the test that is removed here failing before this change and passing afterwards? And if you provide the extra rule, with a `#TODO` linking it to the issue in K, do all the tests pass?